microsoft_intune/plugin.spec.yaml

```
@@ -614,4 +614,40 @@ actions:
         title: Success
         description: Return true if action was successfully performed on device
         type: boolean
-        required: true
\ No newline at end of file
+        required: true
+  add_app_to_policy:
+    title: Add application to a policy
```

Title style this, will need to regen after - and make sure it's reflected in help.md

```suggestion
    title: Add Application to a Policy
```